repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/publication.ftl

```diff
@@ -227,7 +229,7 @@
                                 <div data-uipath="ps.publication.resources-attachments">
                                     <#list document.attachments as attachment>
                                         <div class="nhsd-!t-margin-top-4" data-uipath="ps.publication.resources-attachment">
-                                            <@downloadBlockAsset attachment.text attachment.resource "${attachment.text}" "" attachment.resource.mimeType attachment.resource.length false true />
+                                            <@downloadBlockAsset document.class.name attachment.resource "${attachment.text}" "" attachment.resource.mimeType attachment.resource.length false true />
```

Comment:
Was this intentional?

Reply:
Class name was required here …. which matches with the docTypes  array in imports.ftl